lib/app.js

```diff
@@ -4,13 +4,11 @@ import { Router, Route, hashHistory } from 'react-router';
 import { ListDatasets, ViewDataset }  from './pages';
 import lightVizClient                 from './client';
 
-import vtkWebClientFactory from 'tonic-io/lib/VtkWeb/client/pvw/ClientFactory';
-import VtkWebSmartConnect  from 'tonic-io/lib/VtkWeb/SmartConnect';
+import vtkWebClientFactory from 'paraviewweb/src/IO/WebSocket/ParaViewWebClient';
```

Comment:
I've changed the API of ParaViewWebClient, so you should pull the latest release of ParaViewWeb (1.0.7) and fix your usage of that class.

Here is an example on how to do it right:

import ParaViewWebClient from 'paraviewweb/src/IO/WebSocket/ParaViewWebClient';

var client = ParaViewWebClient.createClient(connection, [
        'FileListing',
        'MouseHandler',
        'ProxyManager',
        'TimeHandler',
        'ViewPort',
    ], customClient);